gensim/models/ldamodel.py

```diff
@@ -900,10 +900,26 @@ def get_document_topics(self, bow, minimum_probability=None):
         if is_corpus:
             return self._apply(corpus)
 
-        gamma, _ = self.inference([bow])
+        gamma, phis = self.inference([bow], collect_sstats=True)
         topic_dist = gamma[0] / sum(gamma[0])  # normalize distribution
-        return [(topicid, topicvalue) for topicid, topicvalue in enumerate(topic_dist)
-                if topicvalue >= minimum_probability]
+
+        if per_word_topics is False:
+            return [(topicid, topicvalue) for topicid, topicvalue in enumerate(topic_dist)
+                    if topicvalue >= minimum_probability]
+
+        elif per_word_topics is True:
+            word_phi = [] # contains word and corresponding topic
+            for word_type, weight in bow:
+                phi_values = [] # contains phi values for each topic
+                for topic_id in range(0, self.num_topics):
+                    if phis[topic_id][word_type] >= minimum_probability:
+                        phi_values.append((topic_id, phis[topic_id][word_type]))
```

Comment:
If we are only doing comparisons for a word within topics and not comparing between words, would this matter?